src/Service/ApiClient.php

```diff
@@ -0,0 +1,50 @@
+<?php
+declare (strict_types=1);
+
+namespace Service;
+
+
+class ApiClient {
+
+    /** @var \Maleficarum\Client\Http\Rest\BasicClient  */
+    protected $httpClient;
+
+    /**
+     * ApiClient constructor.
+     *
+     * @param \Maleficarum\Client\Http\Rest\BasicClient $httpClient
+     */
+    public function __construct(\Maleficarum\Client\Http\Rest\BasicClient $httpClient) {
+        $this->httpClient = $httpClient;
+    }
+
+    /**
+     * @param int $articleId
+     *
+     * @return array
+     */
+    public function getArticle(int $articleId): array {
```

Comment:
Sugeruje call'e przenieść do SDK (w Miinto nazywa się to `Common` jeżeli dobrze ogarnąłem).